src/client/Main.java

```diff
@@ -0,0 +1,36 @@
+package client;
+
+import client.auth.Encryption;
+import client.auth.Authenticator;
+import client.util.InputHandler;
+
+
+
+public class Main {
+    public static void main(String[] args) {
+        try {
+            System.out.println("WELCOME !!!!");
+
+            Authenticator authenticator = new Authenticator();
+
+            System.out.println("Please login !!!!");
+
+            String userId = InputHandler.getStringInput("Enter User ID");
+            Encryption encrypt = new Encryption();
+            String password = InputHandler.getStringInput("Enter the password");
+            String userPassword = encrypt.encrypt(password);
+            
+          
+            
+
+            boolean authenticationStatus = authenticator.authenticate(userId, userPassword);
```

Comment:
Multiple things are happening here. The main class could have taken care of application startup and client initialization. Here, UI interaction is there in the form of messages displayed and user inputs being taken. Then, authentication is also happening. Please handle better.